README.md

```diff
@@ -290,36 +308,37 @@ The attenuations (i.e. UCAN output, or "caveats") MUST be an array of heterogene
 
 This array MUST contain some or none of the following:
 1. A strict subset (attenuation) of the capability scope from the `prf` field
-2. Capabilities composed from multiple witnesses (see rights amplification)
+2. Capabilities composed from multiple proofs (see rights amplification)
 3. Capabilities originated by the `iss` DID (i.e. by parenthood)
 
-This array also includes time ranges and the witnesses that start latest and end soonest from the lower and upper time bounds of the range.
+This array also includes time ranges and the proofs that start latest and end soonest from the lower and upper time bounds of the range.
 
-The attenuation field MUST contain an array of JSON objects, which MAY be empty. A JSON capability MUST include the `with` and `can` fields and MAY have additional fields needed to describe the capability.
+The attenuation field MUST contain an array of JSON objects, which MAY be empty. A JSON capability MUST include the `with` and `can` fields, and MAY additionally have additional fields needed to describe the capability.
```

Comment:
"additional" included twice now: `... MAY additionally have additional fields ...`.
I personally like the idea of changing it into `MAY additionallly have fields...`.

Reply:
But with only two `l`s 😛 Another good catch; thanks @matheus23!